network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/primary_heroguts.html

```diff
@@ -42,7 +42,7 @@
         <div class="col-12 col-md-9 col-xl-6 d-flex flex-column justify-content-end banner-content">
           <h1 class="h1-heading">{{ page.hero_headline }}</h1>
           <div>
-            <a href="{{page.hero_button_url}}" class="btn btn-secondary">{{page.hero_button_text}}</a>
+            <a href="{{page.hero_button_url}}" class="btn btn-secondary text-wrap">{{page.hero_button_text}}</a>
```

Comment:
accidental left-over from the other PR?

Reply:
what! how did this happen 🤷‍♀️ 🤦 . Thanks for catching it.